src/aggregate/expr/expression.c

```diff
@@ -123,6 +124,9 @@ static int getPredicateBoolean(const RSValue *l, const RSValue *r, RSCondition o
 
     default:
       assert("Unknown predicate received" && 0);
+      if (qerr) {
+        QueryError_SetErrorFmt(qerr, QUERY_EEXPR, "Unknown predicate received");
+      }
```

Comment:
@rafie I might have missed something but we have an assert(0) before, this is a dead code.

Reply:
Right.